redisinsight/ui/src/components/navigation-menu/NavigationMenu.tsx

```diff
@@ -203,53 +230,53 @@ const NavigationMenu = () => {
     },
   ]
 
+  const renderNavItem = (nav: INavigations) => (
+    <React.Fragment key={nav.tooltipText}>
+      {renderOnboardingTourWithChild(
+        (
+          <HighlightedFeature
+            {...getAdditionPropsForHighlighting(nav.pageName)}
+            key={nav.tooltipText}
+            isHighlight={!!highlightedPages[nav.pageName]?.length}
+            dotClassName={cx(styles.highlightDot, { [styles.activePage]: nav.isActivePage })}
+            tooltipPosition="right"
+            transformOnHover
+          >
+            <EuiToolTip content={nav.tooltipText} position="right">
+              <div className={styles.navigationButtonWrapper}>
+                <EuiButtonIcon
+                  className={nav.getClassName()}
+                  iconType={nav.getIconType()}
+                  aria-label={nav.ariaLabel}
+                  onClick={nav.onClick}
+                  data-testid={nav.dataTestId}
+                />
+                {nav.isBeta && (<EuiBadge className={styles.betaLabel}>BETA</EuiBadge>)}
+              </div>
+            </EuiToolTip>
+          </HighlightedFeature>
+        ),
+        { options: nav.onboard },
+        nav.isActivePage
+      )}
+    </React.Fragment>
+  )
+
   return (
     <EuiPageSideBar aria-label="Main navigation" className={cx(styles.navigation, 'eui-yScroll')}>
       <div className={styles.container}>
         <EuiToolTip
-          content={server?.buildType === BuildType.RedisStack ? 'Edit database' : 'My Redis databases'}
+          content={server?.buildType === BuildType.RedisStack ? 'Edit database' : isRdiWorkspace ? 'My RDI databases' : 'My Redis databases'}
```

Comment:
Should `'My RDI databases'` be `'My RDI instances'`?

Reply:
yes, thanks